client/my-sites/site-settings/jetpack-credentials/credentials-form/index.jsx

```diff
@@ -78,158 +78,136 @@ export class CredentialsForm extends Component {
+						<FormLabel htmlFor="server-port">{ translate( 'Port Number' ) }</FormLabel>
+						<FormTextInput
+							name="port"
+							id="server-port"
+							placeholder={ translate( '22' ) }
+							value={ get( this.state.form, 'port', '' ) }
+							onChange={ this.handleFieldChange }
+							disabled={ formIsSubmitting }
+							isError={ !! formErrors.port }
+						/>
+						{ formErrors.port && <FormInputValidation isError={ true } text={ formErrors.port } /> }
+					</FormFieldset>
+				</div>
+
+				<div className="credentials-form__row">
+					<FormFieldset className="credentials-form__username">
+						<FormLabel htmlFor="server-username">{ translate( 'Username' ) }</FormLabel>
+						<FormTextInput
+							name="user"
+							id="server-username"
+							placeholder={ translate( 'username' ) }
+							value={ get( this.state.form, 'user', '' ) }
+							onChange={ this.handleFieldChange }
+							disabled={ formIsSubmitting }
+							isError={ !! formErrors.user }
+						/>
+						{ formErrors.user && <FormInputValidation isError={ true } text={ formErrors.user } /> }
+					</FormFieldset>
+
+					<FormFieldset className="credentials-form__password">
+						<FormLabel htmlFor="server-password">{ translate( 'Password' ) }</FormLabel>
+						<FormPasswordInput
+							name="pass"
+							id="server-password"
+							placeholder={ translate( 'password' ) }
+							value={ get( this.state.form, 'pass', '' ) }
+							onChange={ this.handleFieldChange }
+							disabled={ formIsSubmitting }
+							isError={ !! formErrors.pass }
+						/>
+						{ formErrors.pass && <FormInputValidation isError={ true } text={ formErrors.pass } /> }
+					</FormFieldset>
+				</div>
+
+				<FormFieldset>
+					<FormLabel htmlFor="wordpress-path">{ translate( 'Upload Path' ) }</FormLabel>
+					<FormTextInput
+						name="abspath"
+						id="wordpress-path"
+						placeholder={ translate( '/public_html/wordpress-site/' ) }
+						value={ get( this.state.form, 'abspath', '' ) }
+						onChange={ this.handleFieldChange }
+						disabled={ formIsSubmitting }
+						isError={ !! formErrors.abspath }
+					/>
+					{ formErrors.abspath && (
+						<FormInputValidation isError={ true } text={ formErrors.abspath } />
+					) }
+				</FormFieldset>
+
+				<FormFieldset>
+					<FormLabel>{ translate( 'Private Key' ) }</FormLabel>
+					<Button disabled={ formIsSubmitting } onClick={ this.togglePrivateKeyField }>
+						{ showPrivateKeyField
+							? translate( 'Hide Private Key' )
+							: translate( 'Show Private Key' ) }
+					</Button>
+					{ showPrivateKeyField && (
+						<div>
+							<FormTextArea
+								name="kpri"
+								value={ get( this.state.form, 'kpri', '' ) }
+								onChange={ this.handleFieldChange }
+								disabled={ formIsSubmitting }
+								className="credentials-form__private-key"
+							/>
+							<p className="form-setting-explanation">
+								This field is only required if your host uses key based authentication.
```

Comment:
Isn't this string supposed to be translatable?

Reply:
Yes. Fixing.